Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:6
+
+# Screwdriver Store Version
+ARG VERSION=latest
+
+# Create our application directory
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+# Install Screwdriver Executor Queue
+RUN npm install screwdriver-executor-queue-worker@$VERSION
+WORKDIR /usr/src/app/node_modules/screwdriver-executor-queue-worker
```

Comment:
We're setting `WORKDIR` multiple times, each time adding another layer. Is there a path issue if you try to do:

```
RUN mkdir ... && cd /usr/src/app && npm install
```